custom/lib/Auth.php

```diff
@@ -48,23 +53,19 @@ public function __construct(private string $appId, private Configuration $config
      *
      * @return string User ID of the Passage user
      * @throws InvalidArgumentException JWT format is invalid
-     * @throws UnexpectedValueException Could not retrieve sub claim from token
+     * @throws UnexpectedValueException Could not validate aud claim or retrieve sub claim from token
      */
     public function validateJwt(string $jwt): string
     {
-        $jwtSegments = explode('.', $jwt);
-        if (count($jwtSegments) !== 3) {
-            throw new InvalidArgumentException('Invalid JWT format');
-        }
```

Comment:
this is already checked by `JWT::decode` [here](https://github.com/firebase/php-jwt/blob/29fa2ce9e0582cd397711eec1e80c05ce20fabca/src/JWT.php#L107-L110)